packages/sdks/react-sdk/examples/app/index.tsx

```diff
@@ -14,6 +14,7 @@ root.render(
       projectId={process.env.DESCOPE_PROJECT_ID!}
       baseUrl={process.env.DESCOPE_BASE_URL}
       baseStaticUrl={process.env.DESCOPE_BASE_STATIC_URL}
+      sessionTokenViaCookie={{ SameSite: 'Lax' }}
```

Comment:
I think its a good idea leaving react sdk sample to manage session token on local storage and not on cookie, lets revert this